packages/sanity-plugin-nrkno-iframe-preview/README.md

```diff
@@ -0,0 +1,166 @@
+# @nrk/sanity-plugin-nrkno-iframe-preview
+
+Liveupdated iframe preview component for Sanity Studio, compatible with any framework.
+
+Queries are executed by Sanity Studio and passed to the iframe. 
+This has the beneficial in that the render-app in the iframe
+does not need to listen for sanity client updates itself.
+
+![preview.png](./docs/images/preview.png)
+_Figure 1: IFramePreview with desktopMinWidth configured_
+
+The app running in the iframe should use `@nrk/nrkno-iframe-preview-api`, 
+which allows it to specify a GROQ-query for the Studio to execute.
+
+The result of the query will be sent to the iframe whenever the query
+revision matches the studio revision (ie. whenever the data changes).
+
+This means that the render app does not need to listen for changes itself, as
+data flows directly from the Studio via iframe.postMessage.
+
+# Installation
+In Sanity studio project run:
+
+`npx sanity install @nrk/sanity-plugin-nrkno-iframe-preview`
+
+This will install & add the plugin to sanity.json plugin array.
+
+## At-a-glance
+
+* Use IFramePreview component in studio structure view.
+    * Configure with render-app preview-url.
+    * Optionally configure desktopMinWidth.
+* Use [nrkno-sanity-iframe-preview-api](../nrkno-iframe-preview-api/README.md) in the render-app.
+    * Configure a groq-query (or just use document directly from Sanity Studio as is).
+* Enjoy live-updated preview in Studio, with queries executed by the Studio on behalf of the render app.
+
+## Usage
```

Comment:
Syntes godt du kan skrive at koden rett under her legger til en global IFramePreview komponent som vi vises for alle dokumenter (om jeg leser dette riktig)

Reply:
Nei det stemmer ikke. Det er en komponent du kan konfigurere på egenhånd.